src/System.Linq.Expressions/tests/ExceptionHandling/ExceptionHandlingExpressions.cs

```diff
@@ -137,6 +137,108 @@ public void CanCatchAndThrowNonExceptionsInterpreted()
         }
 
         [Fact]
+        public void ThrowNullThrowsNRE()
+        {
+            Action throwNull = Expression.Lambda<Action>(
+                Expression.Throw(Expression.Constant(null, typeof(Expression)))
+                ).Compile(false);
+            Assert.Throws<NullReferenceException>(throwNull);
+        }
+
+        [Fact]
+        public void ThrowNullThrowsNREInterpreted()
+        {
+            Action throwNull = Expression.Lambda<Action>(
+                Expression.Throw(Expression.Constant(null, typeof(Expression)))
+                ).Compile(true);
+            Assert.Throws<NullReferenceException>(throwNull);
+        }
```

Comment:
> There are tests, including in this file, where the one works and one relates to an active issue, which makes it a bit less consistent

That's fine.  I'd rather avoid most of the duplication, and then if one of the cases needs to be made an active issue, just change it to be:
```C#
[ActiveIssue(12345)]
public void Foo_Interpreted()
{
    Foo(compiled: false);
}

[Theory]
[InlineData(true)]
public void Foo(bool compiled)
{
    ...
}
```
Then when the issue is fixed it can be recombined by deleting the ActiveIssue test and just adding an [InlineData(false)] attribute to the real test.